Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,43 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox
+    {
+        public Size Size { get; set; } //default set to 0 in the enum cant be null
```

Comment:
```suggestion
        public Size Size { get; } //default set to 0 in the enum cant be null
```
Theses could be readonly

Reply:
I think All should be able to have a get and a set in case the order changes. But I did make your suggested correction in my Assignment_5 branch